sklearn/model_selection/tests/test_validation.py

```diff
@@ -740,3 +740,37 @@ def test_cross_val_predict_sparse_prediction():
     preds_sparse = cross_val_predict(classif, X_sparse, y_sparse, cv=10)
     preds_sparse = preds_sparse.toarray()
     assert_array_almost_equal(preds_sparse, preds)
+
+
+def test_cross_val_predict_apply_func_cv():
+    iris = load_iris()
+    X, y = iris.data, iris.target
+    classes = len(set(y))
+
+    # Shuffle the data before doing cross validation
+    shf = np.random.permutation(np.arange(len(X)))
```

Comment:
tests shouldn't use an arbitrary random state. You can use `X, y = sklearn.utils.shuffle(X, y, random_state=0)`